cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -125,7 +125,7 @@
  */
 public class Capacity {
     protected static final ReconciliationLogger LOGGER = ReconciliationLogger.create(Capacity.class.getName());
-
+    protected final Reconciliation reconciliation;
```

Comment:
Do you need it to be protected instead of private?

Reply:
It does not, just changed